### PR TITLE
Merge master with feature/GraphingCalculator

### PR DESCRIPTION
**Pull request by sanderl**
### Description of the changes:
- Merge with master to get latest calculator changes into the feature/GraphingCalculator
-
-

### How changes were validated:

- Used PRs to find missing code
- Manual smoke testing
-

### CODE REVIEW

File: src/Calculator/Calculator.vcxproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <AppxDefaultResourceQualifierUAP_Contrast>black</AppxDefaultResourceQualifierUAP_Contrast>
     <AppxBundle>Always</AppxBundle>
-    <PackageCertificateKeyFile>TemporaryKey.pfx</PackageCertificateKeyFile>
+    <!--<PackageCertificateKeyFile>TemporaryKey.pfx</PackageCertificateKeyFile>-->
```
Comment (joseartrivera) [inline src/Calculator/Calculator.vcxproj]: We probably don't want this checked in

**Comment by joseartrivera** [comment]
I repro this error when building the UT project locally:

error LNK2022: metadata operation failed (80131187) : Inconsistent method declarations in duplicated types...

I don't repro this in master, nor in feature/GraphingCalculator. Seems specific to this branch. I took some time to try and figure out why but I couldn't find the cause.

**Comment by sanderl** [comment]
> I repro this error when building the UT project locally:
> 
> error LNK2022: metadata operation failed (80131187) : Inconsistent method declarations in duplicated types...
> 
> I don't repro this in master, nor in feature/GraphingCalculator. Seems specific to this branch. I took some time to try and figure out why but I couldn't find the cause.

I resolved this issue. It was due to the target sdk being mismatched. The unit tests were pointing to 18362 while the other projects were updated to 19019.

File: src/CalcViewModel/Common/KeyboardShortcutManager.cpp [hunk 2]
```diff
@@ -71,7 +71,7 @@ namespace CalculatorApp
             }
```
Comment (joseartrivera) [inline src/CalcViewModel/Common/KeyboardShortcutManager.cpp]: the 1+1=3 bug is still reproing, looks like it was caused by a bad merge in #585. To fix we will want to remove the extra call to RunFirstEnabledButtonCommand(buttons) on line 574.

File: src/CalcViewModel/Common/KeyboardShortcutManager.cpp [hunk 3]
```diff
@@ -83,31 +83,15 @@ namespace CalculatorApp
...
-                // Cancel the timer after we're done so it only fires once
-                auto timer = timerWeakReference.Resolve<DispatcherTimer>();
-                if (timer)
-                {
-                    timer->Stop();
-                }
-            });
-            timer->Start();
+            co_await winrt::resume_background();
+            co_await winrt::resume_after(500ms);
```
Comment (joseartrivera) [inline src/CalcViewModel/Common/KeyboardShortcutManager.cpp]: Not related to this PR, but we should consider reverting this change or altering the ms here. To me the keypad feels laggier with this change, but I could see the argument that it makes it clearer visually which key was activated.

**Comment by joseartrivera** [comment]
Calculator seems to be crashing frequently on this branch when interacting with the graphing mode, might just be my computer though...

File: src/Calculator/App.xaml [hunk 4]
```diff
@@ -18,18 +18,33 @@
...
+                                     Opacity="0.4"
+                                     Color="{ThemeResource SystemAltMediumLowColor}"/>
                     <SolidColorBrush x:Key="AppControlPageTextBaseMediumHighBrush" Color="{StaticResource SystemBaseMediumHighColor}"/>
-                    <RevealBackgroundBrush x:Key="AppControlHoverButtonFaceBrush" Color="#18FFFFFF"/>
-                    <RevealBackgroundBrush x:Key="AppControlPressedButtonFaceBrush" Color="#30FFFFFF"/>
+                    <RevealBackgroundBrush x:Key="AppControlHoverButtonFaceBrush" Color="#18FFFFFF" FallbackColor="#18FFFFFF"/>
+                    <RevealBackgroundBrush x:Key="AppControlPressedButtonFaceBrush" Color="#30FFFFFF" FallbackColor="#30FFFFFF"/>
                     <SolidColorBrush x:Key="AppControlTransparentAccentColorBrush" Color="{ThemeResource SystemAccentColor}"/>
                     <SolidColorBrush x:Key="AppControlPageTextBaseHighColorBrush" Color="{StaticResource SystemBaseHighColor}"/>
-                    <SolidColorBrush x:Key="AppControlForegroundAccentBrush" Color="{ThemeResource SystemAccentColor}"/>
```
Comment (joseartrivera) [inline src/Calculator/App.xaml]: This resource is being referenced in GraphingCalculator.xaml, which is why I keep seeing crashes in graphing mode.

**Pull request closed without merging**